tools/build.py

```diff
@@ -91,6 +91,12 @@
                       default=False,
                       help="Compile the DSP library")
 
+    parser.add_argument("-F", "--fat",
```

Comment:
this is good for testing, but I would not send this to master now. The reason is that this fat file system is targeting mbed-os, and soon enough the new rtos will come that wont be supported in these scripts (use mbed-cli). Thus any tools/ changes in this PR shall be reverted. What do you think?

cc @sg-